### PR TITLE
Fill engagement activity trend with zeros if null is returned for type.

### PR DESCRIPTION
**Pull request by dsjen**
@rocha @clintonb 

FYI: @shnayder 

Before:
before

After:
after

### CODE REVIEW

File: analytics_dashboard/courses/presenters.py [hunk 1]
```diff
@@ -64,9 +64,11 @@ def _build_trend(self, api_trends):
         for datum in api_trends:
             trend_week = {'weekEnding': self.strip_time(datum['interval_end'])}
             for trend_type in trend_types:
-                trend_week[trend_type] = datum[trend_type] if trend_type in datum else 0
+                trend_week[trend_type] = datum[trend_type] if datum[trend_type] else 0
             trends.append(trend_week)
 
+        print trends
```
Comment (clintonb) [inline analytics_dashboard/courses/presenters.py]: Remove the print.

File: analytics_dashboard/courses/presenters.py [hunk 2]
```diff
@@ -64,9 +64,11 @@ def _build_trend(self, api_trends):
         for datum in api_trends:
             trend_week = {'weekEnding': self.strip_time(datum['interval_end'])}
             for trend_type in trend_types:
-                trend_week[trend_type] = datum[trend_type] if trend_type in datum else 0
+                trend_week[trend_type] = datum[trend_type] if datum[trend_type] else 0
```
Comment (clintonb) [inline analytics_dashboard/courses/presenters.py]: This can be simplified to: `trend_week[trend_type] = datum[trend_type] or 0`.

Comment (dsjen) [inline analytics_dashboard/courses/presenters.py]: That's interesting.  Thanks!

**Comment by dsjen** [comment]
Updated

**Comment by clintonb** [comment]
:+1:

**Comment by rocha** [comment]
Any of you know why we are getting errors on the CI tests?

**Pull request merged**